testing/util/deneb.go

```diff
@@ -91,7 +147,10 @@ func GenerateTestDenebBlobSidecar(t *testing.T, root [32]byte, header *ethpb.Sig
 		KzgCommitment:     commitment,
 		KzgProof:          commitment,
 	}
-	pb.CommitmentInclusionProof = fakeEmptyProof(t, pb)
+	if incProof == nil {
```

Comment:
I did this because there are some existing uses of `GenerateTestDenebBlobSidecar` outside this package that I didn't want to update.